src/test/java/justin/ToDoTest.java

```diff
@@ -0,0 +1,20 @@
+package justin;
+
+import justin.task.ToDo;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ToDoTest {
+    @Test
+    public void dummyTest1() {
```

Comment:
Could name your unit testing in a more suggestive way? Like valid/ invalid and should return what kind of output